homeassistant/components/pyload/config_flow.py

```diff
@@ -0,0 +1,154 @@
+"""Config flow for pyLoad integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from aiohttp import CookieJar
+from pyloadapi.api import PyLoadAPI
+from pyloadapi.exceptions import CannotConnect, InvalidAuth, ParserError
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_NAME,
+    CONF_PASSWORD,
+    CONF_PORT,
+    CONF_SSL,
+    CONF_USERNAME,
+    CONF_VERIFY_SSL,
+)
+from homeassistant.core import DOMAIN as HOMEASSISTANT_DOMAIN, HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+from homeassistant.helpers.aiohttp_client import async_create_clientsession
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.issue_registry import IssueSeverity, async_create_issue
+
+from .const import DEFAULT_HOST, DEFAULT_NAME, DEFAULT_PORT, DOMAIN, ISSUE_PLACEHOLDER
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_NAME, description={"suggested_value": DEFAULT_NAME}): str,
+        vol.Required(
+            CONF_HOST, description={"suggested_value": "homeassistant.local"}
+        ): str,
```

Comment:
there's a helper for suggested values. don't know the name exactly

Reply:
hmm, i got this from here: https://developers.home-assistant.io/docs/data_entry_flow_index#defaults--suggestions

Reply:
you already use the helper btw, but only for user_input

Reply:
as far as i understand this used in combination, wen the form is called if there is no `user_input` the `seggested_values` defined in the schema are used to prefill the form, otherwise, the values from user_input will be used. Or am i wrong?

Reply:
instead of putting suggested_values like this you can do user_input or {...<the values you have here...}

Reply:
From the description of `add_suggested_values_to_schema`:

> For each schema marker matching items in suggested_values, the suggested_value will be set. The existing suggested_value will be left untouched if there is no matching item.

Using `user_input or` means all or nothing. It doesn't make any difference here, because all fields are required but in other cases it would be a difference in functionality